examples/simple/README.md

```diff
@@ -0,0 +1,82 @@
+# Jupyter Server Simple Extension Example
+
+This folder contains an example of 2 simple extensions on top of Jupyter Server.
+
+## Install
+
+You need `python3` to build and run the server extensions.
+
+```bash
+conda create -y -n jext python=3.7
+conda activate jext
+make build
```

Comment:
I think the `make` file is convenient for aliasing commands in this example, but I'm afraid it might obscure what's actually happening at each step. 

I think we should either 1) drop the Make file and use the actual commands or 2) clearly state of the top of this document that readers should refer the make file for a more direct understanding of what's happening. 

This is particularly necessary for the more subtle commands, like:
```
jupyter server --ServerApp.jpserver_extensions="{'simple_ext1': True, 'simple_ext2': True}"
```

Since we don't have this documented anywhere else, it would be nice to showcase this command in your examples here.